AirBoard/AirBoard/View Model/AirportViewModel.swift

```diff
@@ -17,85 +17,88 @@ class AirportViewModel {
     // MARK: Properties
     weak var delegate: AirportsViewModelDelegate?
     
-    private var airports: [Airport]
-    var airportsToDisplay: [String: [Airport]]
-    var sectionTitles: [String]
+    var data = [String: [Airport]]() {
+        didSet {
+            delegate?.reciveData()
```

Comment:
you can read about this observers in this short article: 
https://medium.com/the-andela-way/property-observers-didset-and-willset-in-swift-4-c3730f26b1e9